### ISSUE TITLE
Enhancement Request: Let the user sort/filter post or custom_posts based on category attribute

### ISSUE THREAD

**Issue by mmarj** [weDevsOfficial/wp-user-frontend#1071]
**Description & Current Behaviour:** 

With our [existing feature](https://wedevs.com/docs/user-dashboard/), it's saying that we can pass attribute like `'form_id' ,  'category' , 'featured_image' , 'meta'  or 'excerpt'`. The existing category attribute will show the output with [assigned categories]( http://prntscr.com/104yr1a). 

But it would be more robust and useful if we let the users sort the posts based on the category with a shortcode like  ``[wpuf_dashboard post_type="product" sort_category="category1"]` which will only show the "category1" posts on the output.

**Use cases:**

Let's say users just want to show a specific category's product on the frontend dashboard so they can use a shortcode like:
`[wpuf_dashboard post_type="product" sort_category="category1"]

Say for example- I have three post categories (category1, category2, category3). Now I want to show only `category1 ` posts on a specific page and  `category2 ` on another page.

Thanks.

**Comment by NicoSerranoP** [comment]
Thanks @mmarj ! That feature would be really useful for all :D

**Comment by NicoSerranoP** [comment]
Guys! I found a solution using an existing shortcode. I would like to comment it in case somebody finds this issue and has a similar problem. My problem was:

I created three post forms (one for category 1, other for category 2 and another for category 3). I wanted my users to have a specific page to see a specific category dashboard. The shortcode [[wpuf_dashboard form_id="99" post_type="product"]](https://wedevs.com/docs/user-dashboard/) allows you to show a dashboard for all posts created with form 99.
